src/omniperf_base.py

```diff
@@ -127,7 +130,14 @@ def detect_profiler(self):
         else:
             #TODO: Add detection logic for rocprofv2
             rocprof_cmd = detect_rocprof()
-            self.__profiler_mode = "rocprofv1"
+            if str(rocprof_cmd).endswith("rocprof"):
+                self.__profiler_mode = "rocprofv1"
+            elif str(rocprof_cmd).endswith("rocprofv2"):
+                self.__profiler_mode = "rocprofv2"
+            else:
+                error("Incompatible profiler. Please review documentation.")
```

Comment:
Consider adapting more useful error message with supported profilers as in src/omniperf_profile/profiler_base.py.

Reply:
Done
https://github.com/AMDResearch/omniperf/blob/cbbd4fe75eeda296e5e4b083a5019a8a399e12e5/src/omniperf_base.py#L134-L140